src/renderer/components/+apps-releases/releases.tsx

```diff
@@ -26,14 +26,14 @@ import kebabCase from "lodash/kebabCase";
 import { disposeOnUnmount, observer } from "mobx-react";
 import type { RouteComponentProps } from "react-router";
 import { releaseStore } from "./release.store";
-import { IReleaseRouteParams, releaseURL } from "./release.route";
 import type { HelmRelease } from "../../api/endpoints/helm-releases.api";
 import { ReleaseDetails } from "./release-details";
 import { ReleaseRollbackDialog } from "./release-rollback-dialog";
 import { navigation } from "../../navigation";
 import { ItemListLayout } from "../item-object-list/item-list-layout";
 import { HelmReleaseMenu } from "./release-menu";
 import { secretsStore } from "../+config-secrets/secrets.store";
+import { ReleaseRouteParams, releaseURL } from "../../../common/routes";
```

Comment:
[nit] Seems to use this pattern in other files
```suggestion
import type { ReleaseRouteParams } from "../../../common/routes";
import { releaseURL } from "../../../common/routes";
```